src/features/organizations/pages/PublicEventPage.tsx

```diff
@@ -219,7 +220,14 @@ const ContactPersonSection: FC<{
   contactPerson: { email?: string; id: number; name: string; phone?: string };
 }> = ({ contactPerson }) => {
   const [expandContactMethods, setExpandContactMethods] = useState(false);
-
+  const user = useUser()
+  if (user?.id===contactPerson.id){
```

Comment:
This is not a reliable way of checking this. "Users" are not the same as "Persons" in Zetkin. A Person can exist in Zetkin without having a user account. User accounts are _connected to_ instances of the person model in an organization, but they are separate entities.

Confusingly, on the development server, most users and the person records that they are connected to have the same sequential ID.

To figure out if the current user is connected to the _person_ that is the contact person in an event, you need more logic. There's some information in the issue on how to do it, and there are other places in the codebase where similar things happens, for example in the "Settings" part.

![image](https://github.com/user-attachments/assets/b59a977c-d588-4383-8b4f-687d2e7d221c)